packages/camera/camera/README.md

```diff
@@ -19,7 +19,12 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
 
 ## Installation
 
-First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.dev/using-packages/).
+First, add `camera` as a
+dependency in your `pubspec.yaml` file:
+
+```bash
+flutter pub add camera
+```
```

Comment:
I've just been removing these blocks as I come across them; pub.dev has a prominent tab with installation instructions. So the old line can be deleted without replacement.

Same throughout the PR.